src/pyopenweathermap/client/free_client.py

```diff
@@ -14,6 +16,10 @@ def __init__(self, api_key, api_type, units="metric", lang='en'):
         self.api_type = api_type
         self.units = units
         self.lang = lang
+        if api_type == 'current':
```

Comment:
let's don't mix different clients together. air pollution client should be created in `owm_client_factory.py`